component-samples/demo/aio/aio.env

```diff
@@ -7,6 +7,7 @@ aio_database_driver=org.h2.Driver
 aio_database_allow_others=false
 aio_database_init_sql=./init.sql
 aio_database_new_device_sql=./new-device.sql
+owner_to0_rv_blob=http://localhost:8042?ipaddress=127.0.0.1
```

Comment:
port can be 8080 instead 8042 as AIO is run on 8080

Reply:
Done.